client/lib/webauthn/index.js

```diff
@@ -183,7 +184,77 @@ function register() {
 			}
 		} );
 }
+function wpcomLoginRequest( action, form ) {
+	form.append( 'client_id', config( 'wpcom_signup_id' ) );
+	form.append( 'client_secret', config( 'wpcom_signup_key' ) );
+	if ( process.env.NODE_ENV === 'development' ) {
+		form.append( 'dev_hostname', window.location.hostname );
+	}
+	return fetch( `https://wordpress.com/wp-login.php?action=${ action }`, {
+		method: 'POST',
+		body: form,
+		credentials: 'include',
+	} ).then( response => response.json() );
+}
+
+function authenticate( wpcom_user_id, nonce ) {
```

Comment:
@xknown:  It's not obvious to me where this is called (even though I definitely see this call going out.  Is it implicit somewhere or am I just too tired to find it :)

Reply:
The `authenticate` method isn't not used anymore :), it was "deprecated" in 99a9581. We can probably remove it

Reply:
Removed it in 8141ef4